tests/test_product.py

```diff
@@ -1,16 +1,29 @@
 import datetime
 
 from mock import Mock
+import pytest
 
 from django.core.urlresolvers import reverse
+from django.utils.encoding import smart_text
 
 from saleor.cart.models import Cart
 from saleor.cart import CartStatus, utils
 from saleor.product import models
+from saleor.product.models.utils import get_attributes_display_map
 from saleor.product.utils import get_availability
 from tests.utils import filter_products_by_attribute
 
 
+@pytest.fixture()
+def product_with_no_attributes(db):
+    product_class = models.ProductClass.objects.create(
```

Comment:
Is there a fixture for the product class in `conftest`? If not, should there be?